db/migrate/20150820234102_add_column_to_packages2.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToPackages2 < ActiveRecord::Migration
```

Comment:
A handy way to avoid having to say "1", "2", etc in the class- / filename is to use more details about what the migration does. This also makes it easier to figure out the changes when you are running them. So this, for example, could be `AddPriceAndServiceTypeToPackages`